src/main/java/com/exactpro/th2/crawler/Crawler.java

```diff
@@ -477,4 +513,12 @@ private FetchIntervalReport(Interval interval, long sleepTime, boolean processFr
             this.processFromStart = processFromStart;
         }
     }
+
```

Comment:
What do you need this class for? I don't see any constructor calls

Reply:
There are a few of `FetchIntervalReport` constructor calls in `getOrCreateInterval` and `createAndStoreInterval` method.

Reply:
Yes, but I was talking about `IntervalStartReport` class